src/components/Pages/Search/Search.js

```diff
@@ -1,7 +1,102 @@
+import {
+  TextField,
+  ThemeProvider,
+  Button,
+  createMuiTheme,
+  Tabs,
+  Tab,
+} from "@material-ui/core";
+import { useState, useEffect } from "react";
+import SearchIcon from "@material-ui/icons/Search";
+import CustomPagination from "../../Pagination/CustomPagination";
+import SingleContent from "../../SingleContent/SingleContent";
+import "./Search.css";
+
 const Search = () => {
+  const [type, setType] = useState(0);
+  const [page, setPage] = useState(1);
+  const [searchText, setSearchText] = useState("");
+  const [content, setContent] = useState();
+  const [numOfPages, setNumOfPages] = useState();
+
+  const darkTheme = createMuiTheme({
+    palette: {
+      type: "dark",
+      primary: {
+        main: "#fff",
+      },
+    },
+  });
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/search/${
+        type ? "tv" : "movie"
+      }?api_key=1e37d8deae00c9b519356f5d9412edbb&language=en-US&query=${searchText}&page=${page}&include_adult=false`
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        setContent(data.results);
+        setNumOfPages(data.total_pages);
+      });
+  }, [searchText, page, type]);
+
   return (
-    <div>
-      <span className="pageTitle">Search</span>
+    <div className="ThemeProvider">
+      <ThemeProvider theme={darkTheme}>
```

Comment:
As stated [in the documentation](https://material-ui.com/styles/api/#themeprovider) components that provide a context should be implemented "at the root of your component tree", which would be in `App.js` for example. That way you won't have to add it several times, but only once. The current theme can then be changed at a high-up level and will always be consistent throughout your whole app.

Reply:
I changed this part yesterday with Namir. The Themeprovider is already in the App.js file.